src/note/note.go

```diff
@@ -0,0 +1,93 @@
+package note
+
+import (
+	"errors"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/skycoin/skycoin/src/cipher"
+	"github.com/skycoin/skycoin/src/util/file"
+)
+
+// Error wraps note-related errors.
+// It wraps errors caused by user input, but not errors caused by programmer input or internal issues.
+type Error struct {
+	error
+}
+
+// NewError creates an Error
+func NewError(err error) error {
+	if err == nil {
+		return nil
+	}
+	return Error{err}
+}
+
+var (
+	// ErrNoteAPIDisabled is returned when trying to do note actions while the EnableNoteAPI option is false
+	ErrNoteAPIDisabled = NewError(errors.New("wallet api is disabled"))
+	// ErrNoteNotExist is returned if a note does not exist
+	ErrNoteNotExist = NewError(errors.New("note doesn't exist"))
+	// ErrInvalidTxID is returned if a transaction ID is not valud
+	ErrInvalidTxID = NewError(errors.New("invalid transaction ID"))
+)
+
+const (
+	// NoteExt file extension
+	NoteExt = ".txnote"
+)
+
+// LoadNotes Loads all notes contained in notes dir. If any regular file in notes
+// dir fails to load, loading is aborted and error returned. Only files with
+// extension NoteExt are considered.
+func LoadNotes(dir string) (map[string]string, error) {
```

Comment:
I think there should only be one notes file. Not sure if we should allow multiple and merge them or not.